src/System.Security.Cryptography.Csp/src/System/Security/Cryptography/CapiHelper.Windows.cs

```diff
@@ -282,19 +283,19 @@ internal static SafeProvHandle CreateProvHandle(CspParameters parameters, bool r
         {
             SafeProvHandle safeProvHandle;
             uint flag = 0;
-            uint hr = unchecked((uint)OpenCSP(parameters, flag, out safeProvHandle));
+            int hr = OpenCSP(parameters, flag, out safeProvHandle);
             //Open container failed 
             if (hr != S_OK)
             {
                 safeProvHandle.Dispose();
                 // If UseExistingKey flag is used and the key container does not exist
                 // throw an exception without attempting to create the container.
                 if (IsFlagBitSet((uint)parameters.Flags, (uint)CspProviderFlags.UseExistingKey) ||
-                                                        ((hr != (uint)CryptKeyError.NTE_KEYSET_NOT_DEF && hr !=
-                                                        (uint)CryptKeyError.NTE_BAD_KEYSET && hr !=
-                                                        (uint)CryptKeyError.NTE_FILENOTFOUND)))
+                                                        ((hr != unchecked((int)CryptKeyError.NTE_KEYSET_NOT_DEF)
```

Comment:
Why are you changing this?  It seems to me to just be adding noise... particularly since you aren't using the `hr` variable in any of the new code.